androidtv/basetv/basetv.py

```diff
@@ -472,6 +472,29 @@ def _running_apps(running_apps_response):
 
         return None
 
+    @staticmethod
+    def _installed_apps(installed_apps_response):
+        """Get the installed apps output of :py:const:`androidtv.constants.CMD_INSTALLED_APPS`.
+
+        Parameters
+        ----------
+        installed_apps_response : str, None
+            The output of :py:const:`androidtv.constants.CMD_INSTALLED_APPS`
+
+        Returns
+        -------
+        list, None
+            A list of the installed apps, or ``None`` if it could not be determined
+
+        """
+
+        if installed_apps_response is not None:
+            if isinstance(installed_apps_response, list):
+                return [line.strip().rsplit("package:", 1)[-1] for line in installed_apps_response if line.strip()]
+            return [line.strip().rsplit("package:", 1)[-1] for line in installed_apps_response.splitlines() if line.strip()]
+
+        return None
```

Comment:
Lines 493 and 496 here aren't covered with tests.  I don't think lines 492 and 493 are necessary, but please add a test that covers line 496.

Reply:
Tests added for all of them